documentation/mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: Jenkins 2.0 Pipelines
 nav:
     - Home: index.md
-    - 'Guided Tour' : guidedtour.md
+    - 'Getting Started With Project "Piper"' : guidedtour.md
```

Comment:
Would it make sense to rename the `guidedtour.md` to something like  `gettingStarted` in order to keep some consistency and pattern between the entry in the toc and the files behind it. Would simplify the file retrieval. I mean can also be merged like it is, but maybe renaming the file would be even better.

Reply:
Ok, that reasonable. There's a couple more references to the guidedtour in md linknames, and its mostly link labels, which I won't change.

https://github.com/SAP/jenkins-library/search?q=guidedtour&unscoped_q=guidedtour